### ISSUE TITLE
Please publish to NuGet

### ISSUE THREAD

**Issue by GoogleCodeExporter** [blyzer/morelinq#66]
```
It would be great to be able to reference this from NuGet.
```

Original issue reported on code.google.com by `mr.fer...@gmail.com` on 22 Apr 2011 at 12:42

**Comment by GoogleCodeExporter** [comment]
Original comment by `azizatif` on 26 May 2011 at 7:45
- Changed state: **Accepted**
- Added labels: **Component-Scripts**, **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
http://nuget.org/List/Packages/morelinq
```

Original comment by `daniel.p...@gmail.com` on 26 Sep 2011 at 2:52

**Comment by GoogleCodeExporter** [comment]
Original comment by `azizatif` on 25 May 2012 at 4:57
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 75fec8063566.
```

Original comment by `azizatif` on 9 Nov 2012 at 9:27
- Changed state: **Fixed**
